package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "world-countries-capitals",
-	"version": "3.1.0",
+	"version": "3.2.1",
```

Comment:
Why is this change within that _Pull Request_? Can we revert that from here?

@bhatvikrant is there any issue with package versioning over _NPM_? Was last release happened properly?

Reply:
I think it got staged somehow and pushed, i will rather add it in gitignore so that it doesn't comes up in my next PR.

Reply:
No there is no issue with versioning, @jav7zaid lets not update the package versions in PRs. I ll update versions while publishing to NPM